LEGO1/mxdirectx/mxdirect3d.cpp

```diff
@@ -734,7 +929,7 @@ int MxDeviceEnumerate::FUN_1009d0d0()
 	int i = 0;
 	int j = 0;
 	int k = -1;
-	unsigned int und = FUN_1009d1a0();
+	unsigned int cpu_mmx = SupportsMMX();
```

Comment:
`cpu_mmx` should probably be an int, or does that negatively affect the accuracy?

Reply:
Well, I did not change this function because I assumes it was already matching.